.github/workflows/release-major-tag.yml

```
@@ -10,6 +10,7 @@ on:
 
 permissions:
   contents: write
+  workflows: write
```

actionlint (1.7.12)

[error] unknown permission scope "workflows". all available permission scopes are "actions", "artifact-metadata", "attestations", "checks", "contents", "deployments", "discussions", "id-token", "issues", "models", "packages", "pages", "pull-requests", "repository-projects", "security-events", "statuses"

(permissions)